crates/env/Cargo.toml

```diff
@@ -65,3 +65,4 @@ std = [
 # Enable contract debug messages via `debug_print!` and `debug_println!`.
 ink-debug = []
 ink-experimental-engine = ["ink_engine"]
+wee-alloc = ["ink_allocator/wee-alloc"]
```

Comment:
What's the plan for `wee-alloc`? I mean, why even keep it in here? If there is some valuable trade-off between both of them we should add some info on it to the docs: https://paritytech.github.io/ink-docs/datastructures/dynamic-allocation.

Reply:
The tradeoff here is `.wasm` size vs. efficient use of memory.`wee_alloc` is able to re-use freed memory, while the `bump` allocator is not. 

I agree though, if we do keep both it would be valuable to document the trade-offs. I've opened https://github.com/paritytech/ink-docs/issues/24.